pkg/cli/klothomain.go

```diff
@@ -298,6 +293,20 @@ func (km KlothoMain) run(cmd *cobra.Command, args []string) (err error) {
 		return nil
 	}
 
+	// Needs to go after the --version and --update checks
+	claims, err := km.Authorizer.Authorize()
+	if claims != nil {
+		analyticsClient.AttachAuthorizations(claims)
```

Comment:
so if we dont have claims because the login failed, what happens here? Will the telemtry just show an email or distinct user id of ""?

Reply:
It should show up with a `localId` uuid that's saved in the credentials.json. For some reason, I can't find any of this telemetry when I run it locally. I suspect that's a me-problem, but in any case I've added it to the test plan to verify.